rs/cross-chain/proposal-cli/BUILD.bazel

```diff
@@ -46,7 +46,7 @@ rust_test(
     name = "unit_tests",
     crate = ":make_proposal",
     data = [
-        "//rs/bitcoin/ckbtc/kyt:kyt.did",
```

Comment:
Do we maybe still want to keep the old KYT around to be able in a few weeks/months to easily retire the old KYT canister, which would require an NNS proposal?

Reply:
Does proposal-cli offer uninstall? I think if it is infrequent enough, maybe just manually write a proposal and use ic-admin

Reply:
> Does proposal-cli offer uninstall?

Good point it doesn't. I agree it's a one-off thing, so not worth it to integrate to proposal-cli.